.travis.yml

```diff
@@ -1,3 +1,5 @@
+%YAML 1.1
```

Comment:
It isn't strictly needed for Travis-CI, but I included it for clarity. The most recent release of the YAML specification is version 1.2, but Travis-CI only supports version 1.1 . Explicitly declaring it just removes slight uncertainty about the allowed YAML syntax. I looked at it sort of like the XML tag at the top of an XML document- it might parse without it but it is more clear to include it. If you still don't want it though I can remove it.